src/components/Footer.tsx

```diff
@@ -4,7 +4,7 @@ import { BULogo } from '../assets/assets.index';
 export const MainFooter: React.FC = () => {
   return (
     <div style={{ width: '100vw%' }}>
-      <img src={BULogo} style={{ display: 'block', marginLeft: 'auto', padding: '20px' }} />
+      <img src={BULogo} style={{ display: 'flex', alignItems: 'left', padding: '20px', width: '648px' }} />
```

Comment:
Can you switch this from inline style to using the corresponding `.less` file? In addition, switch from `px` to something responsive. `rem`, `em` would probably be a good choice. `vh,vw` could also work well.

Reply:
Sure Ian,  I'll make these changes and commit them.